src/CommandingOfficers/Commander.java

```diff
@@ -226,7 +228,13 @@ public GameEventQueue initTurn(GameMap map)
       aiController.initTurn(myView);
     }
 
-    return new GameEventQueue();
+    GameEventQueue events = new GameEventQueue();
```

Comment:
Why move unit init here?

One difference (which may or may not matter) is that units will now register events all at once instead of sequentially.

Reply:
Makes more sense, to me, to have it here, and it added unnecessary complexity to MapController IMO.

That difference would matter if the UnitHealEvent pre-calculated anything, but it doesn't, so it doesn't.

Reply:
I guess it would also matter if someone listened to UnitHealEvents and published events afterwards. That'd be weird though.